Rakefile

```diff
@@ -70,7 +70,19 @@ namespace :test do
   desc "available tests are: [:test:all, :test:local, :test_integration, :test_resources]"
   task :all => [:local, :integration, :resources]
 
-## default unit tests
+  ## The "current" test template is available to allow a place to specify tests
+  ## of current interest during development.  The files specified are whatever
+  ## is convenient at the moment.
+  Rake::TestTask.new do |t|
+    t.libs << "test"
+    t.name = "current"
+    t.description = "current tests of interest"
+    #    t.test_files = FileList['**/test_WAPI.rb']
```

Comment:
remove this line?

Reply:
It's good to have a sample line.  I added some clarifications to the text in the Rakefile.